README.md

```diff
@@ -66,6 +66,24 @@
 
 ## Installation
 
+### Swift Package
+The [Swift Package Manager](https://swift.org/package-manager/ "") is a tool for automating the distribution of Swift code and is integrated into the swift compiler.
+
+Once you have your Swift package set up, adding `web3swift` as a dependency is as easy as adding it to the `dependencies` value of your `Package.swift`.
+```swift
+dependencies: [
+    .package(url: "https://github.com/web3swift-team/web3swift.git", .upToNextMajor(from: "3.0.0"))
+]
+```
+
+## Example usage
+In the imports section:
+
+```swift
+import web3swift
+import Web3Core
+```
+
 ### CocoaPods
```

Comment:
Should we add here a sentence right away about current issues or are they already being fixed? @yaroslavyaroslav

Reply:
Worth adding something if building with CocoaPods doesn't work.
If it does, probably the warning on the bottom is good enough. Depends on the severity of the issue as I do not recall what was the last state for the CocoaPods.

Reply:
@janndriessen These can be added later. In the current state, PR looks good to me and I suggest we merge it.

Thanks @VaishaliDesai!

Reply:
Sure, thank you @JeneaVranceanu. I think github has blocked the merging until we get one more approval from @janndriessen

Reply:
@JeneaVranceanu it doesn't, but i'm planning to fix it at least in someway with the 3.1.1 release.